draft-ietf-ppm-dap.md

```diff
@@ -804,19 +804,42 @@ the privacy guarantee is not affected by the sampling rate of the population,
 therefore a larger than expected batch size does not weaken the designed privacy
 guarantee.
 
-Implementation note: The goal for the Aggregators is to aggregate precisely
-`min_batch_size` reports per batch. Doing so, however, may be challenging for
-Leader deployments in which multiple, independent nodes running the aggregate
-interaction (see {{aggregate-flow}}) need to be coordinated. The maximum batch
-size is intended to allow room for error. Typically the difference between the
-minimum and maximum batch size will be a small fraction of the target batch size
-for each batch. If `max_batch_size` is not specified, the goal for Aggregators
-is to output the batch once it meets `min_batch_size`. How soon the batch should
-be output is deployment specific.
-
-[OPEN ISSUE: It may be feasible to require a fixed batch size, i.e.,
-`min_batch_size == max_batch_size`. We should know better once we've had some
-implementation/deployment experience.]
+Implementation note: The goal for the Aggregators is to aggregate the same
+number of reports in each batch. The target batch size is deployment-specific,
+and may be equal to or greater than the minimum batch size. Deciding how soon
+batches should be output is also deployment-specific. Exactly sizing batches may
+be challenging for Leader deployments in which multiple, independent nodes
+running the aggregate interaction (see {{aggregate-flow}}) need to be
+coordinated. The difference between the minimum batch size and maximum batch
+size is in part intended to allow room for error, and allow a range of target
+batch sizes.
+
+### Batch Size Considerations {#batch-size}
+
+If each batch will be collected only once (i.e. when using Prio3), then batch
+sizes may be equal to `min_batch_size` without issue. In the case of fixed size
+queries, `max_batch_size` may be equal to `min_batch_size`.
+
+If each batch may be collected more than once (i.e. when using Poplar1), then
+batches should be larger than `min_batch_size`, to allow for the possibility
+that some reports may successfully pass through VDAF preparation with the first
+aggregation parameter, but fail with a subsequent aggregation parameter. Once a
```

Comment:
Instead of "pass/fail preparation", I prefer the verbiage "accept/reject".
```suggestion
that some reports may be accepted with the first
aggregation parameter, but be rejected with a subsequent aggregation parameter. Once a
```